erpnext/stock/doctype/item/item.py

```diff
@@ -634,9 +636,23 @@ def validate_name_with_item_group(self):
 				_("An Item Group exists with same name, please change the item name or rename the item group"))
 
 	def update_item_price(self):
-		frappe.db.sql("""update `tabItem Price` set item_name=%s,
-			item_description=%s, brand=%s where item_code=%s""",
-					(self.item_name, self.description, self.brand, self.name))
+		frappe.db.sql("""
+				UPDATE `tabItem Price`
+				SET
+					item_name=%(item_name)s,
+					item_description=%(item_description)s,
+					brand=%(brand)s,
+					uom=%(uom)s
```

Comment:
```suggestion
```

Item prices can have different UOMs, also they don't change after the first transaction. No need to update them.